src/managed/CommonManaged.props

```diff
@@ -17,6 +34,21 @@
     <PackageThirdPartyNoticesFile>$(RepoRoot)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(StabilizePackageVersion)' != 'true'">
+    <!-- Dev builds get a minor version, by default, of '0'.  Our tests restore packages with
```

Comment:
Is this still true? I didn't think we are doing * versions any longer so we should just be able to do the defaults here.